src/backend/app/submissions/submission_crud.py

```diff
@@ -524,3 +530,45 @@ async def get_submission_detail(
         odk_form.getSubmissions(project.odkid, db_xform.odk_form_id, submission_id)
     )
     return submission.get("value", [])[0]
+
+
+async def get_submission_geojson(
```

Comment:
I noticed we already have central_crud.convert_odk_submission_json_to_geojson and submission_routes.download_submission_geojson that all basically do the same thing.

Could we refactor to only have the logic once, then reuse the code?

Reply:
if i want geojson only for the conflation, then this function won't work since it returns submission in bytes. like you said, we can refactor the `download_submission` endpoint to use `convert_odk_submission_json_to_geojson`.

Reply:
The second option sounds best in the spirit of DRY code.

A single function to return a geojson, then encode as bytes where specifically needed.

If absolutely necessary it's also possible to decode bytes back to geojson using `.decode("utf--8")` 👍

json.load would probably also do that